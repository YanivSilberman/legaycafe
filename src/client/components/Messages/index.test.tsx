import * as React from 'react';
import * as ReactShallowRenderer from 'react-test-renderer/shallow'
import Component from './index';

describe('Messages component test',() => {
  const renderer = ReactShallowRenderer.createRenderer();

  it('component loads', () => {
    expect(renderer.render(
      <Component
        users={{}}
        userId="123"
        waitingOnMessage={false}
      />
    )).toMatchSnapshot()
  })
});
