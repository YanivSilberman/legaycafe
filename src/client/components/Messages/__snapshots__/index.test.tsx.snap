// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages component test component loads 1`] = `
<withApollo(Apollo(Apollo(Messages)))
  classes={
    Object {
      "date": "withApollo(Apollo(Apollo(Messages)))-date-5",
      "isTyping": "withApollo(Apollo(Apollo(Messages)))-isTyping-4",
      "messageContainer": "withApollo(Apollo(Apollo(Messages)))-messageContainer-2",
      "messages": "withApollo(Apollo(Apollo(Messages)))-messages-1",
      "ownMessageContainer": "withApollo(Apollo(Apollo(Messages)))-ownMessageContainer-3",
      "progress": "withApollo(Apollo(Apollo(Messages)))-progress-9",
      "skeletonMessage": "withApollo(Apollo(Apollo(Messages)))-skeletonMessage-7",
      "snack": "withApollo(Apollo(Apollo(Messages)))-snack-8",
      "top": "withApollo(Apollo(Apollo(Messages)))-top-6",
    }
  }
  userId="123"
  users={Object {}}
  waitingOnMessage={false}
/>
`;
